imports/ui/components/sidebar_menu/PanelProfile.jsx

```diff
@@ -16,20 +17,43 @@ export default class PanelProfile extends React.Component {
     console.log(data);
   }
 
+  openExemptedModuleInfoPage(){
+    window.open("http://www.comp.nus.edu.sg/programmes/ug/exemptions/polygrad/", "_blank");
+  }
+
+  openWaivedModuleInfoPage(){
+    window.open("http://www.comp.nus.edu.sg/programmes/ug/cs/curr/#_ftnref7", "_blank");
+  }
+
   render() {
+    let exemptedToolTip =
```

Comment:
Awesome! I will be referring to your example here about making this tooltip for each Module in the Study Plan dashboard!!